spec/Section 7 -- Response.md

```diff
@@ -53,6 +52,24 @@ the following JSON concepts:
 | Float         | Number            |
 | Enum Value    | String            |
 
+**Object Property Ordering**
+
+While JSON Objects are specified as an
+[unordered collection of key-value pairs](https://tools.ietf.org/html/rfc7159#section-4)
+the pairs are represented in an ordered manner. In other words, while the JSON
+strings `{ "name": "Mark", "age": 30 }` and `{ "age": 30, "name": "Mark" }`
+encode the same value, they also have observably different property orderings.
+While not every JSON generator allows for control over the order in which
```

Comment:
I might remove this bit, and do something like:

  From the above, the spec notes that "Serialization formats which can represent an ordered map should preserve the order of requested fields as defined by query execution." Hence, if the query was `{ name, age }`, a GraphQL server vending JSON should respond with `{ "name": "Mark", "age": 30 }`, preserving the order of the requested field in the JSON serialization.